src/cmake/boost.cmake

```diff
@@ -41,6 +41,8 @@ if (NOT Boost_FOUND)
   ##    Helpful if there are conflicts between locally and system-installed versions
   set (Boost_NO_BOOST_CMAKE OFF)
 
+set(BOOST_ROOT "/usr/include")  
```

Comment:
Unless I did this it would keep getting overwritten by something, my inexperience with cmake meant I couldn't figure out exactly what was doing it but this change solved the issues

Reply:
It all looks ok to me!

Actually, in my case, I actually install Boost from source.  That is what I had been doing for many years.  And that made the `README.md` specific to me (i.e., the export of `BOOST_ROOT` to `/usr/local/boost_1_59_0/` is a good example of this -- it was pointing to a local installation).

I don't have a better idea than what you proposed.  I certainly don't think everyone should install Boost from source...it is truly painful (but I've done it so many times, it's "easy" for me).

Thank you for the PR!